client/webserver/site/src/css/market.scss

```diff
@@ -427,6 +427,12 @@ table.balance-table button:hover {
   }
 }
 
+div.numorders {
+  color: white;
+  background-color: rgb(94, 94, 130);
+  border-radius: 3px;
+}
```

Comment:
The coloring is a little bright. It should blend in and match existing styles more closely.

![image](https://user-images.githubusercontent.com/6109680/124758565-4ff86f00-def4-11eb-9d4a-b0ba9a3a4d0c.png)

To do that, I added `line-height: 1; padding: 1px;` here, and then removed the `px-1` class and added the `align-items-center` class to the wrapper `div`. You might want to play with `font-size: 0.9em` too, but the font-size is already pretty small here so I'm not sure we want to push it.